.github/workflows/verify_library_generation.yaml

```diff
@@ -21,7 +21,9 @@ jobs:
       run: |
         set -ex
         git checkout "${base_ref}"
+        git fetch --no-tags --prune origin +${base_ref}:refs/remotes/origin/${base_ref}
```

Comment:
Why `git fetch` after `git checkout`?

Reply:
It doesn't actually matter too much for L23-L24, but it's needed for L25/26 so that it can find the forked branches. I can revert this line if preferred.

Reply:
If it's not actually useful, I think we should revert it.

Reply:
Actually I think we do need it. Added some comments to explain.